### PR TITLE
Ta/i like purple butterflies

### PR DESCRIPTION
**Pull request by taranguest**
I think this it is? Not completely sure I have pushed my branch back to the repo? Cool learnings!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# learn-react
\ No newline at end of file
+# I __love__ *REACT*
```
Comment (lauchness) [inline README.md]: No need to change the original file

File: README.md.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+#I __love__ *REACT*
```
Comment (lauchness) [inline README.md.txt]: So this is good, but the file shouldn't be a `.txt`, just a `.md`. Also, put it in a subfolder. Once you're done that, commit your local changes and push them back up here.

Nice work!

**Comment by lauchness** [comment]
So, you need to make your changes to this branch, and push them back up here. That way I can review them in this PR. Once they're resolved, we'll merge this PR.

**Comment by taranguest** [comment]
I think I made the changes to the actual branch now - removed the .txt from
the file and added a sub folder called TA. Took some playing around and
tried to delete the unnecessary PRs. Is this still attached to the
initial PR?

File: README.md.txt (hunk 2 above)
Comment (lauchness) [reply README.md.txt]: This file still exists, it should be deleted

File: README.md [hunk 3]
```diff
@@ -1 +0,0 @@
-# learn-react
```
Comment (lauchness) [inline README.md]: Looks like you deleted the original README.md, leave this one alone

Comment (lauchness) [reply README.md]: i.e., put it back the way it was so there is no diff in your PR

**Comment by lauchness** [comment]
Ok, we're getting there. Only one thing left, you've made changes to the original README.md (you deleted it). 

You can undo those changes by finding the commit hash that removed the file OR you can just make the file again and make sure it matches what's in master on this repo.

File: README.md (hunk 3 above)
Comment (lauchness) [reply README.md]: Ok, so the file is back, but you can see here that there's a diff.

The red shows what used to be in the file, and the green shows what's being changed by your PR

**Pull request merged**